CMakeLists.txt

```diff
@@ -327,7 +322,9 @@ message(STATUS "  Using syslog.h .................. : ${HAS_SYSLOG}")
 endif()
 message(STATUS "  TelnetAppender .................. : ${LOG4CXX_NETWORKING_SUPPORT}")
 message(STATUS "  NTEventLogAppender .............. : ${HAS_NT_EVENTLOG_APPENDER}")
+if(WIN32)
```

Comment:
should this be moved up one line?  The NTEventLogAppender would only be available on Windows.

Reply:
On the basis that showing an unavailable option is not helpful, yes NTEventLogAppender status should only be shown on WIN32